app/forms/new_phone_form.rb

```diff
@@ -155,6 +165,7 @@ def ingest_submitted_params(params)
     @otp_delivery_preference = delivery_prefs if delivery_prefs
     @otp_make_default_number = true if default_prefs
     @recaptcha_token = params[:recaptcha_token]
+    @recaptcha_version = 2 if params[:recaptcha_version].to_i == 2
```

Comment:
is there a reason we have to hardcode it like this and not let recaptcha version be passed in?

Reply:
The main idea was to limit the set of potential values we'd allow for constructing the `RecaptchaValidator` class, since we wouldn't want to allow invalid versions, e.g. `4`, etc ([related spec](https://github.com/18F/identity-idp/pull/7826/files#diff-d615dba1d9e4b6a922fa2953d329fcf9a88719b014a442e2655437e3002a2fd1R402-R410)). Perhaps that should be handled a bit more strictly within the validator class itself, though.

Reply:
I think I'd like to keep this in the controller, but I also added some extra checks to the validator class in e4f0e1ac6 to make sure that it's initialized correctly.